- adjust version requirements
- remove "unimportant" tree-sitter-* dependencies that aren't yet ported

--- pyproject.toml.orig	2025-07-08 06:36:22 UTC
+++ pyproject.toml
@@ -1,7 +1,7 @@ build-backend = "setuptools.build_meta"
 [build-system]
 build-backend = "setuptools.build_meta"
 
-requires = [ "cython>=3.0.12", "setuptools>=80.9.0", "typing-extensions>=4.14.0" ]
+requires = [ "cython", "setuptools", "typing-extensions>=4.14.0" ]
 
 [project]
 name = "tree-sitter-language-pack"
@@ -37,9 +37,9 @@ dependencies = [
 ]
 dependencies = [
   "tree-sitter>=0.23.2",
-  "tree-sitter-c-sharp>=0.23.1",
-  "tree-sitter-embedded-template>=0.23.2",
-  "tree-sitter-yaml>=0.7.0",
+  #"tree-sitter-c-sharp>=0.23.1",
+  #"tree-sitter-embedded-template>=0.23.2",
+  #"tree-sitter-yaml>=0.7.0",
 ]
 
 urls.documentation = "https://github.com/Goldziher/tree-sitter-language-pack#readme"
